microsoft-edge/playwright/index.md

```diff
@@ -11,110 +11,130 @@ keywords: microsoft edge, web development, developer, tools, automation, test, p
+When using the above `playwright.config.ts` file, Playwright Test uses Microsoft Edge to run your tests, as follows:
 
-*   Chromium  `await chromium.launch()`
-*   Firefox  `await firefox.launch()`
-*   WebKit  `await webkit.launch()`
-
-For more information about Playwright, navigate to the [Playwright website][PlaywrightMain].  Check out the  [Playwright repo][PlaywrightRepo] on GitHub.  To share your feedback on automating and testing your website or app with Playwright, [file an issue][PlaywrightRepoNewIssue].
+```shell
+npx playwright test --headed
+```
 
 
 <!-- ====================================================================== -->
-<!-- links -->
+## Playwright as a library
 
-[WebdriverChromiumMain]: ../webdriver-chromium/index.md "WebDriver | Microsoft Docs"
-[PuppeteerMain]: ../puppeteer/index.md "Puppeteer | Microsoft Docs"
+You can also consume Playwright as a library, as shown in the following code.  This approach allows you to use a different test-runner.
 
-[MicrosoftBlogsWindowsExperience20181206]: https://blogs.windows.com/windowsexperience/2018/12/06/microsoft-edge-making-the-web-better-through-more-open-source-collaboration "Microsoft Edge: Making the web better through more open-source collaboration | Microsoft Experience Blog"
+```javascript
+// example.js
+const playwright = require('playwright');
 
-[MicrosoftEdgeDownload]: https://microsoft.com/edge "Download Microsoft Edge"
+(async () => {
+  const browser = await playwright.chromium.launch({
+    channel: 'msedge-beta',
+  });
+  const context = await browser.newContext();
+  const page = await context.newPage();
+  await page.goto('https://www.microsoft.com/edge');
+  await page.screenshot({ path: 'example.png' });
 
-[ChromiumHome]: https://www.chromium.org/Home "Chromium | The Chromium Projects"
+  await browser.close();
+})();
+```
 
-[FirefoxMain]: https://www.mozilla.org/firefox "Mozilla Firefox"
+:::image type="content" source="../media/playwright-example.png" alt-text="The example.png file produced by example.js." lightbox="../media/playwright-example.png":::
 
-[NodejsMain]: https://nodejs.org "Node.js"
+`example.js` is a simple demonstration of the automation and testing scenarios that are enabled by Playwright.  To take screenshots in other web browsers, change the above code from `await playwright.chromium.launch` to the following code:
 
-[npmEdgePaths]: https://www.npmjs.com/package/edge-paths "edge-paths | npm"
+Firefox: 
 
-[PlaywrightMain]: https://playwright.dev "Playwright"
-[PlaywrightAPIReference]: https://playwright.dev#?path=docs/api.md "Playwright API Reference"
-[PlaywrightAPIPageSetViewport]: https://playwright.dev#?path=docs%2Fapi.md&q=pagesetviewportsizeviewportsize "page.setViewportSize(viewportSize) | Playwright API Reference"
-[PlaywrightSystemRequirements]: https://playwright.dev#?path=docs/intro.md&q=system-requirements "Playwright System Requirements"
+```javascript
```

Comment:
@mxschmitt - please tech-review this code and the WebKit code below it.

Reply:
looks good!